submissions/maryOkosun/dommanipulation/newsletter.js

```diff
@@ -0,0 +1,28 @@
+let newsForm = document.getElementById('newsection'),
+	newsletter = document.getElementById('newsletter');
+
+let mailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/; //eslint-disable-line
+
+/*FORM VALIDATION*/
+	
+function emailValidation(newsletter,event){
```

Comment:
Since you are validating for just email, there is no need to passing newsletter as a parameter here. Simply use the email field direct

Reply:
@adaugochi 
Thanks a lot.
I just effected the corrections.
Please review.